services/app-api/src/emailer/awsSES.ts

```diff
@@ -74,7 +74,6 @@ async function sendSESEmail(
         console.info('SendEmailCommand: ', command)
         return await ses.send(command)
     } catch (err) {
-        console.error(JSON.stringify(err))
         return err
```

Comment:
Wait.. I'm a little confused now. I thought we were moving to throwing errors from here and not continuing to return them?

Reply:
Oops forgot this. Too eager to get this up for PR.